### DISCUSSION TITLE
rtsp 4k to rtsp FHD

### DISCUSSION THREAD

**Discussion by Arit1900** [bluenviron/mediamtx#513] [Questions]
Hi,
Now i'm running rtsp 4k to rtsp 4k, can i reduce it to FHD?

**Comment by aler9** [comment]
You can perform any compression you want by using ffmpeg together with rtsp-simple-server, like described here:

https://github.com/aler9/rtsp-simple-server#remuxing-re-encoding-compression

In particular, changing resolution is one of the easiest things to do.

[a comment by an automated account is not shown]
